datasets/cn/peoples_congress_wikipedia/crawler.py

```diff
@@ -119,6 +161,60 @@ def parse_table(
         yield row
 
 
+def parse_table_with_rowspan(
+    context: Context,
+    table: HtmlElement,
+) -> Generator[Dict[str, HtmlElement], None, None]:
+    headers = None
+    rowspan_cells = {}  # Dictionary to keep track of cells with rowspan
```

Comment:
I think it'd be good to document a data structure underpinning something like this - it looks like keys are column indices and values are tuples where the first element is what? and the second element is the span value?